src/routes/formsg/formsgSiteAuditLogs.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/prefer-default-export */
+import type { DecryptedContentAndAttachments } from "@opengovsg/formsg-sdk/dist/types"
+import express, { RequestHandler } from "express"
+
+import { config } from "@root/config/config"
+import InitializationError from "@root/errors/InitializationError"
+import logger from "@root/logger/logger"
+import { attachFormSGHandler } from "@root/middleware"
+import AuditLogsService from "@root/services/admin/AuditLogsService"
+import { getField, getFieldsFromTable } from "@root/utils/formsg-utils"
+
+interface FormsgSiteAuditLogsRouterProps {
+  auditLogsService: AuditLogsService
+}
+
+const SITE_AUDIT_LOGS_FORM_KEY = config.get("formSg.siteAuditLogsFormKey")
+
+const REQUESTER_EMAIL_FIELD = "Where should we send the email address to?"
+const REPO_NAME_FIELD =
+  "What is the name of the Isomer site that you need logs for? (Repo Name (in GitHub))"
+const LOGS_TIMEFRAME_FIELD = "I need a log of edits made in:"
+const LOGS_TIMEFRAME_START_FIELD = "Start date"
+const LOGS_TIMEFRAME_END_FIELD = "End date"
+
+export class FormsgSiteAuditLogsRouter {
+  private readonly auditLogsService: FormsgSiteAuditLogsRouterProps["auditLogsService"]
+
+  constructor({ auditLogsService }: FormsgSiteAuditLogsRouterProps) {
+    this.auditLogsService = auditLogsService
+  }
+
+  getAuditLogsHandler: RequestHandler<
+    never,
+    Record<string, never>,
+    { data: { submissionId: string } },
+    never,
+    { submission: DecryptedContentAndAttachments }
+  > = async (req, res) => {
+    let startDate = "1970-01-01"
```

Comment:
Hmm I'm not sure if 6 years ago would be arbitrary here as compared to using epoch, which would be clearer in the intent?

Reply:
oh its not an arbitrary number, its when isomer was born haha

Reply:
This one I might want to push back. I don't have an example but setting it to say 2018-01-01 means we may miss potential commits that occur before that date, which can happen if for some reason users "backdate" their commits. I try to avoid making this decision for users just in case, especially this is audit logs that needs as much information captured as possible.

Reply:
sure, lgtm